public/app/themes/justice/src/patch/js/ccfw-gtm.js

```diff
@@ -188,6 +188,14 @@ const CCFW = {
  * @returns {boolean}
  */
 const init = () => {
+    /**
+     * We cannot run if we don't have a storage object
+     * Run a test to check the Storage engine
+     */
+    if (Object.hasOwn(Storage, 'disabled')) {
+        return false;
+    }
```

Comment:
Another minor, I suppose this could potentially introduce a bug further down the line, where another developer may stick `disabled: false` on the Storage object - this conditional would be true.

It would be a bit safer to explicitly check if disabled is true here.

Reply:
It's not the best option but definitely progress. `disabled` won't exist on the Storage object in most cases. I've added a note above the object declaration to help in this circumstance. Thanks for pointing it out.